partner_invoicing_mode_monthly/models/res_config_settings.py

```diff
@@ -0,0 +1,14 @@
+# Copyright 2020 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html)
+from odoo import fields, models
+
+
+class ResConfigSettings(models.TransientModel):
+    _inherit = "res.config.settings"
+
+    res_invoicing_mode_monthly_day_todo = fields.Integer(
+        related="company_id.invoicing_mode_monthly_day_todo", readonly=False
+    )
+    invoicing_mode_monthly_last_execution = fields.Datetime(
```

Comment:
```suggestion
    res_invoicing_mode_monthly_last_execution = fields.Datetime(
```

to keep same convention for both fields in this file?

Reply:
thanks. I have updated